kano_updater/ui/media/flappy-judoka/bin/flappy-judoka

```diff
@@ -76,6 +91,12 @@ def main():
     debugger('flappy-judoka: main: Exiting')
     pygame.quit()
 
+    # letting the tracking session know the game is closing
+    try:
+        session_end(tracking_session)
+    except:
+        debugger('ERROR: flappy-judoka: main: Ending tracking session failed!')
```

Comment:
Does the debugger log it somewhere? It'd be good to get logs if this will be failing.

I got burned a few times with ignoring all exceptions -- it makes it really difficult to find the error if there's one.